### PR TITLE
util/bitmap: migrate test-infra to testify

### LINKED ISSUE #26168: migrate test-infra to testify for `util/bitmap` pkg

### PR DESCRIPTION
**Pull request by ateb14**
## What problem does this PR solve?

Issue Number: close #26168 

## Release note

- No release note.

### CODE REVIEW

File: util/bitmap/concurrent_test.go [hunk 1]
```diff
@@ -14,22 +14,17 @@
 package bitmap
 
 import (
+	"github.com/stretchr/testify/assert"
 	"sync"
 	"sync/atomic"
 	"testing"
-
-	. "github.com/pingcap/check"
 )
```
Comment (tisonkun) [inline util/bitmap/concurrent_test.go:21]: These imports violate the goimports rule. If you use GoLand, you may config as
image

**Review by tisonkun — CHANGES_REQUESTED** [review]
Thanks for your contribution @ateb14 ! Clean work!

A few comments inline.

File: util/bitmap/concurrent_test.go (hunk 1 above)
Comment (tisonkun) [reply util/bitmap/concurrent_test.go:21]: Also check_dev failed on `go fmt`. You can run `go fmt ./...` from the root path of tidb repo and submit the change set.

File: util/bitmap/concurrent_test.go [hunk 2]
```diff
@@ -81,7 +77,6 @@ func (s *testBitmap) TestConcurrentBitmapUniqueSetter(c *C) {
 		}
 	}
 	wg.Wait()
-	// If clearCounter is too big, it means setter concurrency of this test is not enough.
-	c.Assert(clearCounter < loopCount, Equals, true)
-	c.Assert(setterCounter, Equals, clearCounter+1)
+	assert.Equal(t, clearCounter < loopCount, true)
```
Comment (tisonkun) [inline util/bitmap/concurrent_test.go:80]:
```suggestion
	assert.Less(t, clearCounter, loopCount)
```

**Review by tisonkun — APPROVED** [review]
LGTM.

Minor suggestion you may integrate.

File: util/bitmap/concurrent_test.go (hunk 1 above)
Comment (ateb14) [reply util/bitmap/concurrent_test.go]: Updated

File: util/bitmap/concurrent_test.go (hunk 2 above)
Comment (ateb14) [reply util/bitmap/concurrent_test.go]: Updated.

**Review by tiancaiamao — APPROVED** [review]

**Review by xhebox — APPROVED** [review]

**Pull request merged**